src/main/java/com/sforce/cd/apexUnit/client/connection/ConnectionHandler.java

```diff
@@ -80,7 +82,14 @@ private ConnectionHandler() {
 				}
 			}
 			SUPPORTED_VERSION = prop.getProperty("API_VERSION");
+			MAX_TIME_OUT_IN_MS = prop.getProperty("MAX_TIME_OUT_IN_MS");
 		}
+		try {
+			MAX_TIME_OUT_IN_MS_INT = Integer.parseInt(MAX_TIME_OUT_IN_MS);
+		} catch (NumberFormatException nfe) {
```

Comment:
If your goal is to restrict it to positive integers you should use relevant exception handler. NumberFormatException scope is beyond +ve numbers